tests/test_client.py

```diff
@@ -1,20 +1,27 @@
 import os
 import json
-
 import requests_mock
 import unittest
 
 import percy
+from percy import config
 
 
 class TestPercyClient(unittest.TestCase):
 
     def setUp(self):
         os.environ['PERCY_TOKEN'] = 'abcd1234'
-        self.percy_client = percy.Client()
+        self.percy_client = percy.Client(config=config.Config(default_widths=(1024, 768)))
```

Comment:
Nitpick: change `(1024, 768)` to `(1280, 375)` for consistency. They are actually just width breakpoints, but using 1024 and 768 makes it looks like a screen size.